### PR TITLE
Add multiplicity marker to replace menu for participants

### LINKED ISSUE #533: I can model a pool with MI marker
In BPMN it is possible to mark a pool as multi instance (at least the marker looks like the parallel multi instance marker)

Should work for both, collapsed and expanded pools

### PR DESCRIPTION
**Pull request by azeghers**
__Which issue does this PR address?__

Closes #533

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js
Comment (MaxTru) [inline test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:203]: The 
```
      // given
      var participant = elementRegistry.get('Participant_1');

      openPopup(participant);
```

is always the same for all 4 test cases isn't it? I suggest to therefore make it part of an "forEach" block. (also applies to the old tests above probably)

**Review by MaxTru — COMMENTED** [review]
Good job.
I tested in locally, runs well.

Only a minor comment.

File: lib/features/popup-menu/ReplaceMenuProvider.js
Comment (pinussilvestrus) [inline lib/features/popup-menu/ReplaceMenuProvider.js:541]: I think we have to be careful with this. [According to the spec, this is **not** a boolean](https://github.com/bpmn-io/bpmn-moddle/blob/master/resources/bpmn/json/bpmn.json#L1970), but an element with type `bpmn: ParticipantMultiplicity`. If we store it is as a boolean, we will get in trouble while serializing (`modeler#saveXML`). 

E.g. when using the inside Camunda Modeler, I got the following error when switching to the XML Editor.

```text
/Users/niklas.kiefer/Desktop/diagram_1.bpmn] Cannot read property 'isGeneric' of undefined
    at ElementSerializer.build (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1226:37)
    at eval (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1472:30)
    at forEach (webpack-internal:///../../bpmn-js/node_modules/min-dash/dist/index.esm.js:181:16)
    at eval (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1460:63)
    at forEach (webpack-internal:///../../bpmn-js/node_modules/min-dash/dist/index.esm.js:181:16)
    at ElementSerializer.parseContainments (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1432:59)
    at ElementSerializer.build (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1247:10)
    at eval (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1472:30)
    at forEach (webpack-internal:///../../bpmn-js/node_modules/min-dash/dist/index.esm.js:181:16)
    at eval (webpack-internal:///../../bpmn-js/node_modules/moddle-xml/dist/index.esm.js:1460:63) [ error ]
```

That's because `moddle` does not know what due to the Boolean value.

Comment (pinussilvestrus) [reply lib/features/popup-menu/ReplaceMenuProvider.js:541]: I think better would be something like

```js
var isActive = entry.active;

var participantMultiplicity;

if(isActive) {
  participantMultiplicity = bpmnFactory.create('bpmn:ParticipantMultiplicity');
}

self._modeling.updateProperties(
  element,
  { participantMultiplicity: participantMultiplicity });
);

```

Comment (pinussilvestrus) [reply lib/features/popup-menu/ReplaceMenuProvider.js:541]: [The `BpmnRenderer`](https://github.com/bpmn-io/bpmn-js/blob/5d339676f2e6fb3b095185631d76c83cdaebca95/lib/draw/BpmnRenderer.js#L1170) is also not checking for a boolean only, but for an _existing_ object.

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]

File: lib/features/popup-menu/ReplaceMenuProvider.js
Comment (pinussilvestrus) [reply lib/features/popup-menu/ReplaceMenuProvider.js:541]: Also, cf. this example diagram: https://github.com/bpmn-io/bpmn-js/blob/master/test/fixtures/bpmn/draw/pools-with-collection-marker.bpmn#L4

**Comment by pinussilvestrus** [comment]
Maybe we can also align the marker with the one for activities. It looks a bit less bold.

image

File: lib/features/popup-menu/ReplaceMenuProvider.js
Comment (azeghers) [reply lib/features/popup-menu/ReplaceMenuProvider.js:541]: Thanks for the catch, I was a bit too quick to implement it.
I'll look into it and update my PR :+1:

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js
Comment (azeghers) [reply test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:203]: I recall @nikku told me a few weeks back that when it came to testing, DRY wasn't the best guideline, as a test was supposed to be understandable as a standalone process, so it made sense to repeat some of the setup for each of them.

File: lib/features/popup-menu/ReplaceMenuProvider.js
Comment (pinussilvestrus) [reply lib/features/popup-menu/ReplaceMenuProvider.js:541]: 🚀 🚀

File: test/spec/features/popup-menu/ReplaceMenuProviderSpec.js
Comment (MaxTru) [reply test/spec/features/popup-menu/ReplaceMenuProviderSpec.js:203]: Well when the goal is understandability, I would still argue that the understandability is not reduced when using a forEach block for this. The key content of the test is to click the icon, and that would still be included in the test hence would be understood.  
Anyway, this is just a suggestion which improves it IMO. But it is also okay the way it is.

**Comment by azeghers** [comment]
Marker aligned and boolean changed to element

image

**Comment by nikku** [comment]
> Marker aligned and boolean changed to element

Great to see all of these on a single screen. Is there a particular reason the vertical offsets are still different? This is not a big one but drives pixel dudes like me crazy :stuck_out_tongue_winking_eye:.

**Comment by azeghers** [comment]
That's how the marker was defined for participants :man_shrugging: 
Do you want me to give it a bit more distance from the bottom?

**Comment by nikku** [comment]
It is fine :see_no_evil: I guess.

**Review by pinussilvestrus — COMMENTED** [review]
Works perfectly fine for me 👏  Good job!

I Would wait for @MaxTru to give his final feedback before merging it.

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**
